packages/smart-contracts/scripts-create2/utils.ts

```diff
@@ -7,15 +7,15 @@ import * as artifacts from '../src/lib';
  * If you want to skip deploying one or more, then comment them out in the list bellow.
  */
 export const create2ContractDeploymentList = [
+  /*  'ChainlinkConversionPath',
```

Comment:
Should you uncomment all those?

Reply:
It's okay to keep it like this imo, when a deployment is required, the list has to be adapted based on what is to be deployed